base_layer/transaction_components/src/consensus/consensus_constants.rs

```diff
@@ -676,7 +686,10 @@ impl ConsensusConstants {
         con3.blockchain_version = BlockVersion::V2;
         con3.valid_blockchain_version_range = 2..=2;
         con3.proof_of_work = algos;
-        let consensus_constants = vec![consensus_constants1, con2, con3];
+        let mut con4 = con3.clone();
+        con4.include_c29_accumulated_difficulty_into_total = true;
+        con4.effective_from_height = 181_000;
```

Comment:
We are already on 180_000; we need to have the seeds deployed before.
```suggestion
        con4.effective_from_height = 185_000;
```